packages/blinks-core/src/api/Action/Action.ts

```diff
@@ -165,6 +165,7 @@ export class Action {
     try {
       return await this._supportStrategy(this, adapter);
     } catch (e) {
+      console.error(e);
```

Comment:
Is this extra error log here necessary? Can we combine it with the existing one?